### DISCUSSION TITLE
How to mark pre-releases as released?

### DISCUSSION THREAD

**Discussion by pinglinh** [intuit/auto#2130] [Q&A]
Hi there, is there a command I can run after `auto shipit` which will get all the pre-release releases and mark them as released?

This is my autorc file:

```json
{
  "plugins": [
    [
      "npm",
      {
        "exact": true,
        "commitNextVersion": true
      }
    ],
    "conventional-commits",
    "first-time-contributor",
    [
      "released",
      {
        "message": "%TYPE went out with version: %VERSION"
      }
    ],
    [
      "jira",
      "https://xxxxxx"
    ]
  ],
  "baseBranch": "main",
  "owner": "xxxx",
  "repo": "xxxx",
  "name": "xxxx",
  "release": {
    "prerelease": true
  }
}
```

**Comment by hipstersmoothie** [comment]
There isn't. "prerelease" here means it's a beta version. So removing the "prerelease" doesn't make sense to me. With auto you just make a real release once you're ready.

**Reply by pinglinh** [reply]
Hmm I want to make a prelease just before code goes out to prod and once it's finished deploying then mark it as 'released'. The issue I have is that it's a monorepo and there could be quite a few prereleases going out all at once so going through them and manually marking them as released would be long.
